packages/esm-patient-banner-app/src/config-schema.ts

```diff
@@ -14,14 +14,24 @@ export const configSchema = {
       _default: [],
     },
   },
-  customAddressLabel: {
-    _type: Type.Object,
-    _description: 'Whether to use custom labels for addresses',
-    _default: {},
+  useCustomAddressLabel: {
+    enabled: {
+      _type: Type.Boolean,
+      _description: 'whether to enable using custom address labels',
+      _default: false,
+    },
+    customAddressLabel: {
+      _type: Type.Object,
+      _description: 'custom labels for addresses',
+      _default: {},
+    },
```

Comment:
@vasharma05 @denniskigen Please let's be a little more thoughtful with config schema, since as you say Vineet, changing things can affect existing implementations.

Having a boolean like this is redundant and obnoxious. We should not punish our implementers like this. Put yourself in the implementer's shoes and think about how they build a config. The only thing a boolean like this does is adds confusion and complexity, breaking things that should work.

Also, the way this has been implemented, the implementer has to provide every piece of the address as a key to the object or else it silently screws up (displays nothing where there should be a label). A better solution is to make it so an implementer only needs to provide configuration keys for the address labels they want to change.

But actually, I don't see why we don't just handle this using translation overrides? These strings go through translation. I suspect this isn't needed at all.